Makefile

```diff
@@ -11,6 +11,8 @@ LDFLAGSSTRING +=-X main.Date=$(BUILD_TIME)
 LDFLAGSSTRING +=-X main.Version=$(GIT_TAG)
 LDFLAGS := -ldflags "$(LDFLAGSSTRING)"
 
+E2EFUZZTEST = FUZZ=true go test ./e2e -fuzz -v -fuzztime=30m
```

Comment:
So this flag was used in the fuzz tests to either skip or not. I think running `go test -v ./...` will run everything and i think it might be needed to skip that.

Reply:
removed it. looks like fuzzer context will be removed.